bodhi/server/models/models.py

```diff
@@ -2233,12 +2233,13 @@ def expire(self):
         if self.expired_date is not None:
             return
 
-        koji_session = buildsys.get_session()
-        try:
-            koji_session.untagBuild(self.build.release.override_tag,
-                                    self.build.nvr, strict=True)
-        except Exception, e:
-            log.error('Unable to untag override %s: %s' % (self.build.nvr, e))
+        if datetime.utcnow() >= self.expiration_date:
```

Comment:
@bowlofeggs I am changing `if datetime.utcnow() >= self.expiration_date:` to `elif self.expired_date is None or datetime.utcnow() >= self.expiration_date:` on the same line which has no failed tests. And I'm also going to make sure to add testcases in the PR :-).